README.md

```diff
@@ -45,12 +45,12 @@ _We want to change that!_
 
 ### 🔌 Officially supported plugins
 
-|                                                                |                                                     |                                                                                    |
-| -------------------------------------------------------------- | --------------------------------------------------- | ---------------------------------------------------------------------------------- |
-| <img width="50" src="docs/images/plugin-eslint.icon.png">      | [ESLint](./packages/plugin-eslint#readme)           | Static analysis using **ESLint** rules.                                            |
-| <img width="50" src="docs/images/plugin-coverage.icon.png">    | [Code Coverage](./packages/plugin-coverage#readme)  | Collects code **coverage** from your tests.                                        |
-| <img width="50" src="docs/images/plugin-js-packages.icon.png"> | [JS Packages](./packages/plugin-js-packages#readme) | Checks 3rd party packages for known **vulnerabilities** and **outdated** versions. |
-| <img width="50" src="docs/images/plugin-lighthouse.icon.png">  | [Lighthouse](./packages/plugin-lighthouse#readme)   | Measures web performance and best practices with **Lighthouse**.                   |
+| Plugin                                                         | Description                                                                                                                             |
+| :------------------------------------------------------------- | :-------------------------------------------------------------------------------------------------------------------------------------- |
+| <img width="50" src="docs/images/plugin-eslint.icon.png">      | [ESLint](./packages/plugin-eslint#readme): Static analysis using **ESLint** rules.                                                      |
+| <img width="50" src="docs/images/plugin-coverage.icon.png">    | [Code Coverage](./packages/plugin-coverage#readme): Collects code **coverage** from your tests.                                         |
+| <img width="50" src="docs/images/plugin-js-packages.icon.png"> | [JS Packages](./packages/plugin-js-packages#readme): Checks 3rd party packages for known **vulnerabilities** and **outdated** versions. |
+| <img width="50" src="docs/images/plugin-lighthouse.icon.png">  | [Lighthouse](./packages/plugin-lighthouse#readme): Measures web performance and best practices with **Lighthouse**.                     |
```

Comment:
I think it makes more sense for the plugin name to be in the _Plugin_ column, not the _Description_ column. We'd need to make the icons much smaller to align the text, but I don't see that as a bad thing. What do you think?

![image](https://github.com/user-attachments/assets/8a8b9b3f-3a8e-486b-8459-f3c2bc6abba1)


```suggestion
| Plugin                                                                                                             | Description                                                                        |
| :----------------------------------------------------------------------------------------------------------------- | :--------------------------------------------------------------------------------- |
| <img width="16" src="docs/images/plugin-eslint.icon.png"> [ESLint](./packages/plugin-eslint#readme)                | Static analysis using **ESLint** rules.                                            |
| <img width="16" src="docs/images/plugin-coverage.icon.png"> [Code Coverage](./packages/plugin-coverage#readme)     | Collects code **coverage** from your tests.                                        |
| <img width="16" src="docs/images/plugin-js-packages.icon.png"> [JS Packages](./packages/plugin-js-packages#readme) | Checks 3rd party packages for known **vulnerabilities** and **outdated** versions. |
| <img width="16" src="docs/images/plugin-lighthouse.icon.png"> [Lighthouse](./packages/plugin-lighthouse#readme)    | Measures web performance and best practices with **Lighthouse**.                   |
```

---

Another alternative would be to have 3 separate columns with distinct column names, allows for larger icons. But I would still reduce the icon size from 50px, it's only for decoration.

![image](https://github.com/user-attachments/assets/57b12c34-0e9d-428c-b343-53b34078908c)


```suggestion
|                              Icon                              | Name                                                | Description                                                                        |
| :------------------------------------------------------------: | :-------------------------------------------------- | ---------------------------------------------------------------------------------- |
|   <img width="24" src="docs/images/plugin-eslint.icon.png">    | [ESLint](./packages/plugin-eslint#readme)           | Static analysis using **ESLint** rules.                                            |
|  <img width="24" src="docs/images/plugin-coverage.icon.png">   | [Code Coverage](./packages/plugin-coverage#readme)  | Collects code **coverage** from your tests.                                        |
| <img width="24" src="docs/images/plugin-js-packages.icon.png"> | [JS Packages](./packages/plugin-js-packages#readme) | Checks 3rd party packages for known **vulnerabilities** and **outdated** versions. |
| <img width="24" src="docs/images/plugin-lighthouse.icon.png">  | [Lighthouse](./packages/plugin-lighthouse#readme)   | Measures web performance and best practices with **Lighthouse**.                   |
```

Reply:
I like the second option 👍 Even when an image is tiny, the content of a markdown table cell is vertically misaligned.